openedx/core/djangoapps/programs/tasks/v1/tasks.py

```diff
@@ -37,6 +36,7 @@ def get_api_client(api_config, student):
     return EdxRestApiClient(api_config.internal_api_url, jwt=id_token)
 
 
+# TODO: Move to program utilities. Will require updating tests.
```

Comment:
@schenedx I might end up taking care of this now. The way I moved imports around is making it difficult to mock a function in this module during testing.